### ISSUE TITLE
Question: Implement a method to generate [BaseColorSchemes]-only themes? 

### ISSUE THREAD

**Issue by oscrom** [batzen/XamlColorSchemeGenerator#3]
I would like to only generate `[BaseColorSchemes].xaml` 

e.g. Light.xaml and Dark.xaml

As of now, I don't have a need for `[BaseColorSchemes].[ColorScheme].xaml`

I have modified the `GenerateColorSchemeFiles` method, adding the following in the foreach loop:

```
if (colorSchemesWithVariantName.Count != 0 || colorSchemesWithoutVariantName.Count != 0) { continue; }
        {
          var themeName = $"{baseColorScheme.Name}";
          this.GenerateColorSchemeFile(outputPath,
                                       templateContent,
                                       themeName,
                                       themeName,
                                       themeName,
                                       string.Empty,
                                       string.Empty,
                                       baseColorScheme.Values,
                                       baseColorScheme.Values,
                                       parameters.DefaultValues);
        }
```

I think a better design would be to overload the method to include a separate flag: `bool generateBaseThemes` instead of only doing so when there are no color schemes.

I can submit a pull request if you would like to add this functionality.

**Comment by batzen** [comment]
I think an overload would only complicate the design as that way `GenerateColorSchemeFiles` would have to make decisions about theme name etc..
I just pushed changes with d4aba482e074857d41a3031e61feff124b0be491 that should match your requirements.

**Comment by batzen** [comment]
And thanks for your suggestion. 
I never thought about that usage scenario and i never thought anyone else, besides ControlzEx, Fluent.Ribbon and MahApps.Metro would use this project. ;-)

**Comment by oscrom** [comment]
Awesome! Thanks for pushing this change.

Heh, yeah, it's a neat pattern to allow designers to make changes without touching much code and just mucking with json values.

**Comment by batzen** [comment]
Did you have any problems integrating it into your build?
I improved that part a bit in the latest versions.

**Comment by oscrom** [comment]
No major problems, I followed MahApps in approach. 

I do think the Using section in the readme could be a bit more clear and explicit in explaining how to use by modifying the .csproj, and maybe some more comments where to define the path. 

For example, in my usage:     

```
<Exec Command="&quot;$(NuGetPackageRoot)\xamlcolorschemegenerator\$(XamlColorSchemeGeneratorVersion)\tools\XamlColorSchemeGenerator.exe&quot; &quot;$(MSBuildProjectDirectory)\Styles\Themes\GeneratorParameters.json&quot;" />
```

It wasn't clear to me that I needed to specify the `GeneratorParameters.json` here from reading the readme.

But looking at the output and comparing with MahApps didn't make it too difficult to figure out what was going on.
